### ISSUE TITLE
The panel text of the calendar applet is misaligned when using a narrow panel

### ISSUE THREAD

**Issue by YorickPeterse** [linuxmint/mint-themes#287]
When using a panel with a narrow height, such as 20, the text of the calendar applet as displayed in the panel is misaligned:

![Screenshot from 2020-07-08 13-19-00](https://user-images.githubusercontent.com/86065/86912692-bd28c700-c11d-11ea-80d0-1c6a738a0230.png)

Increasing panel heights does not appear to _solve_ the issue, but it will become less obvious. For example, this is using a panel height of 30:

![Screenshot from 2020-07-08 13-19-27](https://user-images.githubusercontent.com/86065/86912737-cf0a6a00-c11d-11ea-9c3f-de055879effc.png)

This is using the Mint-Y-Dark theme. For reference, this is what it used to look like in older versions (can't remember exactly which one sadly), and in themes such as Arc Dark:

![Screenshot from 2020-07-08 13-21-36](https://user-images.githubusercontent.com/86065/86912828-f5c8a080-c11d-11ea-844c-a70aebf3c650.png)

Note how the text is aligned to the middle of the workspace applet to the right, instead of being aligned to the bottom.

I tested this using mint-themes 1.8.6 as built from `master`. The version of Cinnamon that I'm running is 4.6.6.

**Comment by YorickPeterse** [comment]
This is more clearly visible in the following image:

![example](https://user-images.githubusercontent.com/86065/86913825-805dcf80-c11f-11ea-874a-412d09994a6d.png)

Here I placed the panel screenshots next to each other, aligned to the bottom of the panels. The red lines are placed to touch the top and bottom of the panel text in the Arc Dark theme. This clearly shows the text of the calendar applet (and perhaps other applets, haven't tried) is aligned to the bottom instead of the middle.

**Comment by smurphos** [comment]
This it the line in the Mint-Y source sass that added the offset to labels - https://github.com/linuxmint/mint-themes/blob/master/src/Mint-Y/cinnamon/sass/_common.scss#L1850

**Comment by smurphos** [comment]
In the complied theme's cinnamon.css

```
  .applet-box StLabel {
    padding-top: 2px; }
```

**Comment by BrunoNZ** [comment]
I commented the line that smurphos pointed out, in _common.scss, recompiled and installed the theme, and it really fixed the bug and i didn't notice anything else broken. It also fixed the notification count position, that i haven't noticed it was buggy until now.

Doing a little research, i found out that the only place that this line "StLabel { padding-top: 2px; }" appears in the compiled files is the one that smurphos also pointed out. Confirming that it doesn't affect anything else.

I'll keep using the fixed theme to test it, but so far so good.

Bugged:
![ThemeAlignment_Bugged](https://user-images.githubusercontent.com/1660320/88324257-d45cdc80-ccf9-11ea-8196-4c563f6a77d6.png)
Fixed:
![ThemeAlignment_Fixed](https://user-images.githubusercontent.com/1660320/88324261-d4f57300-ccf9-11ea-9581-83f9608be193.png)
